cmd/skyeye/main.go

```diff
@@ -53,48 +55,112 @@ var (
 )
 
 func init() {
+	skyeye.Flags().StringVar(&configPath, "config-path", ".", "Path to a config file e.g. '/home/user/xyz'. It is looking for a file called skyeye-config.yaml")
```

Comment:
Instead of the directory and a hardcoded file name and type, have the user pass in the full path to the file. This is handy in a development environment, I can stuff a bunch of config files for different servers in one directory instead of making a separate directory for each one.

You can use SetConfigFile (https://pkg.go.dev/github.com/dvln/viper#SetConfigFile) to do this.

Reply:
Very good input. thx. done